test/IISIntegration.FunctionalTests/IISIntegration.FunctionalTests.csproj

```diff
@@ -14,7 +14,19 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Server.IIS\Microsoft.AspNetCore.Server.IIS.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Server.IISIntegration\Microsoft.AspNetCore.Server.IISIntegration.csproj" />
-    <ProjectReference Include="..\WebSites\**\*.csproj">
+    <ProjectReference Include="..\WebSites\InProcessWebSite\InProcessWebSite.csproj">
```

Comment:
Why expand?

Reply:
msbuild has been automatically expanding it.